dev/install-python-packages.sh

```diff
@@ -2,7 +2,7 @@
 
 python3 -m pip install --upgrade pip
 
-python3 -m pip install pep8 virtualenv
+python3 -m pip install pep8 virtualenv certifi
```

Comment:
this is not needed if you install it in tools/pom.xml

Reply:
Neither actually works. I will have to find out why `setup.py` fails to find the CA certificates. Maybe I will split the macOS changes into separate PR.

Reply:
actually, Travis now says the macOS builds pass which is weird since I saw them failing.

Reply:
They definitely fail. Also, the macOS builds are broken since the recent JDK 9 push. Looks like this changed with more recent Python as we switched from xcode8.3 to 9.3.